.github/workflows/release.yml

```diff
@@ -16,21 +21,28 @@ jobs:
 
   check-release-tag:
 
-    # Only run this job on the main repository and not on forks
-    if: github.repository == 'aiidateam/aiida-core'
+    # Only run this job on the main repository and not on forks.
+    # and only run it if a tag is pushed or if the test-release label is active.
+    # When the tag is released we will publish to pypi.
+    # When the a label is active we will publish to testpypi
```

Comment:
This comment does not really correspond to the if statement below it.

Reply:
Yes the logic is distributed because of the github workflow syntax. I put the comment on top.

Reply:
```suggestion
```